public/javascripts/SVLabel/src/SVLabel/zoom/ZoomControl.js

```diff
@@ -14,7 +14,7 @@ function ZoomControl (canvas, mapService, tracker, uiZoomControl) {
         },
         status = {
             disableZoomIn: false,
-            disableZoomOut: false
+            disableZoomOut: true
```

Comment:
By default the user should not be able to zoom out without first zooming in (at least this is the behavior of our tool on Chrome). This behavior is not replicated in Edge. The ZoomOut button can be clicked, and clicking on it will actually zoom in instead of zoom out.  To fix this, I disable zooming out until the user zooms in. upon which zooming out is enabled.